src/styles/variables.scss

```diff
@@ -0,0 +1,19 @@
+$font-family: Roboto, Arial, Helvetica, sans-serif;
+$color-grey: #eee;
+$color-pink: #ffbfcb;
+$animation-duration: 0.5s;
+$padding-box: 10px;
+$start-day: 1;
+$week-end-day: 7;
+$month-length: 29;
```

Comment:
The month length should be 31, as in your case some of the days are missing
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/003fb5eb-f34d-4472-882e-01b9a1ec0f56)